### DISCUSSION TITLE
There are errors in small details of the Trivy-Docs document

### DISCUSSION THREAD

**Discussion by zangcc** [aquasecurity/trivy#5128] [Documentation]
### Description

Hi trivy👋 ~ this is a small problem I discovered during use.

The parameter in the document is “-ignore-status”, but the correct one is 
“--ignore-status”.

![image](https://github.com/aquasecurity/trivy/assets/64825932/c1f44e09-8320-41c6-a44c-11d2edf23408)

### Link

https://aquasecurity.github.io/trivy/v0.45/docs/configuration/filtering/

### Suggestions

Change it to --ignore-status

**Comment by knqyf263** [comment]
Thanks. https://github.com/aquasecurity/trivy/issues/5129
